react-native-fbsdklogin/js/FBSDKLoginManager.ios.js

```diff
@@ -51,63 +57,52 @@ type FBSDKDefaultAudience = $Enum<{
   'only-me': string,
 }>;
 
-/**
- * Provides methods for logging the user in and out.
- */
-class FBSDKLoginManager {
+module.exports = {
```

Comment:
I can think of *one* reason to keep the thin JS wrapper object for now: adding additional features to the JS part. For instance, in native code, `logInWith{Read|Publish}Permissions` also returns the access token. To do that in react-native-fbsdk, we'd want to create an actual `FBSDKAccessToken` JS object and return that. So that'll require a bit of unpacking and re-packing in the callbacks. See PR #4 for background.